webserver/grpc/src/main/java/io/helidon/webserver/grpc/GrpcConfigBlueprint.java

```diff
@@ -53,12 +57,22 @@ interface GrpcConfigBlueprint extends ProtocolConfig {
 
     /**
      * Whether to support compression if requested by a client. If explicitly
-     * disabled, no compression will be ever be used by the server even if a
+     * disabled, no compression will ever be used by the server even if a
      * client-compatible compressor is found.
      *
      * @return true if compression is enabled
      */
     @Option.Configured
     @Option.DefaultBoolean(true)
     boolean enableCompression();
+
```

Comment:
This will be a backward incompatible change. This can now be worked around, as you can make it a default method, and then add it as a builder method through annotation.

Reply:
Let me see if I can figure out how to do the second part ...

Reply:
`io.helidon.builder.api.Prototype.IncludeDefaultMethods` - you can list names of methods to include.